deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
Automatic rebuilding of what? Doesn't the version bump force a rebuild? I had considered doing this in libosxunwind itself, but that doesn't seem right because this is julia specific. Our build system has plenty of quirks. If packagers mess this up, they'll get an undefined symbol message and if they grep the code base they'll find this file, so I'm not super concerned.